src/shogun/converter/StochasticProximityEmbedding.h

```diff
@@ -135,39 +137,36 @@ class StochasticProximityEmbedding : public EmbeddingConverter
 		int32_t get_max_iteration() const;
 
 		/** get name */
-		virtual const char* get_name() const;
+		const char* get_name() const override;
```

Comment:
I am actually not sure what our policy here is, we used to keep the `virtual`. But it is not needed with `override` I guess.
@gf712 @vigsterkr thoughts?

Reply:
This is the correct way to do it, but I think it causes lots of compiler warnings... because we always use virtual... We should write a libtooling script to replace all the virtual get_name with override, or even just use sed

Reply:
@vinnik-dmitry07 whilst you are at it can you also fix all of these, please?